pkg/event_processor/processor_test.go

```diff
@@ -84,15 +84,35 @@ func TestEventProcessor_Serve(t *testing.T) {
 
 	lines = strings.Split(string(bufString), "\n")
 	ok := true
+	h2Req := 0
+	h2Resq := 0
 	for _, line := range lines {
 		if strings.Contains(strings.ToLower(line), "dump") {
 			t.Log(line)
 			ok = false
 		}
+		// http2 parse error log
+		if strings.Contains(line, "[http2 re") {
+			t.Log(line)
+			ok = false
+		}
+		// http2 parse count
+		if strings.Contains(line, "Name:HTTP2Response") {
+			h2Resq += 1
```

Comment:
Simple counting is not the best method. If a new HTTP/2 binary file is added here, then the numerical verification will fail.

I will submit a new PR to fix this issue.